src/test/java/com/algorand/algosdk/abi/TestMethod.java

```diff
@@ -6,42 +6,67 @@
 import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
nit: I think the old import might still work (but if not, it might be better just to import the new packages individually anyway)